### PR TITLE
RESOLVE Implement repositories back end 

### PR DESCRIPTION
**Pull request by Kevw94**

### CODE REVIEW

**Review by Loule95450 — APPROVED** [review]
Review ok pour moi 👍

File: back-end/src/api/auth/auth.service.ts [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import { LoginDto } from '@/api/auth/dto/auth.dto';
 
 import { User } from '@/api/base/users/interfaces/users.interface';
-import { UsersRepository } from '@/api/base/users/users.repository';
 
 class AuthService {
-	public usersRepository = new UsersRepository();
+	// public usersRepository = new UsersRepository();
```
Comment (Maghwyn) [inline back-end/src/api/auth/auth.service.ts:6]: Mistake or intentional ?

**Review by Maghwyn — APPROVED** [review]
Good for me.
Tho, be careful with the import since we're using alias with typescript.

**Comment by Loule95450** [comment]
Delete ?

**Pull request merged**